src/components/ControlLink/ControlLink.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import classNames from 'classnames';
+import { Clickable, ClickableProps } from '../Clickable';
+
+import './ControlLink.scss';
+
+export type ControlLinkProps = ClickableProps & {
+  color?: 'cement' | 'ink';
```

Comment:
I think there are only 2 versions in the designs, underlined and not underlined, so the color probably shouldn't be customizable.

Reply:
okay